.github/workflows/release.yml

```diff
@@ -0,0 +1,139 @@
+name: Create release
+on: [workflow_dispatch]
+
+jobs:
+  release:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - uses: actions/setup-python@v5
+      with:
+        python-version: "3.8"
+
+    - name: Install Hatch
+      run: pipx install hatch
+
+    - name: Set variables
+      id: set_variables
+      run: |
+        tag=$(git for-each-ref refs/tags --sort=-taggerdate --format='%(refname:strip=2)' --count=1)
```

Comment:
The tags I've created so far are lightweight tags, which don't have a `taggerdate` associated with them. We can change this going forward with however we want to prepare releases, but sorting by `creatordate` instead will work for both lightweight and annotated tags. 

https://git-scm.com/docs/git-for-each-ref

Reply:
done in 33c17c1